.travis.yml

```diff
@@ -5,9 +5,6 @@ language: go
 services:
   - docker
```

Comment:
how are we going to run unit / integration tests that have dependencies on node?

Reply:
I think that is issue #36 if you want everything self-contained.

Right now, all docker commands are available in the travis environment, so if we build the image, we can also use the image. Binary should be able to run and talk to the docker daemon.